repos/spack_repo/builtin/packages/mfem/package.py

```diff
@@ -632,18 +736,18 @@ def find_optional_library(name, prefix):
 
         # Determine C++ standard to use:
         cxxstd = None
-        if self.spec.satisfies("@4.0.0:"):
+        if spec.satisfies("@4.0.0:"):
```

Comment:
I wonder if it would be simpler here and perhaps a more correct expression of mfem's requirements to *default* to `cxxstd = 14` and set something other than that only under specific conditions. I mean, it is still ok as is but am wondering if simplifying would help.